reporters/junit_reporter.go

```diff
@@ -119,8 +121,9 @@ func (reporter *JUnitReporter) SpecDidComplete(specSummary *types.SpecSummary) {
 
 func (reporter *JUnitReporter) SpecSuiteDidEnd(summary *types.SuiteSummary) {
 	reporter.suite.Tests = summary.NumberOfSpecsThatWillBeRun
-	reporter.suite.Time = summary.RunTime.Seconds()
+	reporter.suite.Time = math.Trunc(summary.RunTime.Seconds() * 1000 / 1000)
```

Comment:
The commit message says "cut testsuite timestamp to three digits after the dot", but does this code really do that? The multiplication + division don't change the value, and then `math.Trunc` just throws away all sub-second digits.

Perhaps you meant this:
`math.Trunc(summary.RunTime.Seconds() * 1000) / 1000`

Reply:
Thank you so much for spotting that @pohly, you are absolutely right. I have opened a PR here https://github.com/onsi/ginkgo/pull/521. What do you think?